src/index.html

```diff
@@ -6,9 +6,67 @@
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link
+      rel="preconnect" href="https://fonts.gstatic.com">
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap"
+      rel="stylesheet"
+    >
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="headerContainer">
+      <a href="#" class="homePage">
+        <img src="./images/logo.png"
+        alt="headerLogo"
+        class="headerLogo">
```

Comment:
If you have 3 or more attributes in the tag each of them should be on new line and with identation:
```
<img
  src="./images/logo.png"
  alt="headerLogo"
  class="headerLogo"
>
```